sortinghat/cmd/load.py

```diff
@@ -49,6 +49,9 @@ class Load(Command):
     engine is selected. The parameter '--match-new' can also be used to match
     only those identities that are new on the registry.
 
+    Previous relationships between identities and their enrollments will be
```

Comment:
«Previous relationships between identities and their enrollments will be removed when the option '--reset' is set.»

I would reword it to:

«If the the option '--reset' is set, previous relationships between identities and their enrollments will be removed»